### PR TITLE
Image load

### PR DESCRIPTION
**Pull request by cbalducci**
Implements load action for images
Requires [docker-api v1.27](https://github.com/swipely/docker-api/releases/tag/v1.27.0)

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

File: libraries/helpers_image.rb [hunk 1]
```diff
@@ -93,6 +93,12 @@ def save_image
           Docker::Image.save(repo, destination, connection)
         end
       end
+
+      def load_image
+        with_retries do
+          Docker::Image.load(source, (), connection)
```
Comment (chasebolt) [inline libraries/helpers_image.rb]: i believe rubocop is complaining about your use of (). changing it to {} should clear it up.

Comment (cbalducci) [inline libraries/helpers_image.rb]: Changed as suggested. Thank you @chasebolt

**Comment by chasebolt** [comment]
looks good. just need a signed CLA and I can get this merged in.

**Comment by cbalducci** [comment]
i've already signed the ICLA and also linked my github account to my chef account.. maybe the check needs to be triggered again?

**Comment by chasebolt** [comment]
The commit was done with an email address not linked with your github account so the commit can not be matched up with your signed CLA.

Either associate the email address to your account, or recommit with the email address linked to your github.

Email address in question is cb185188@ncr.com on commit af58c4c

**Comment by chasebolt** [comment]
resolves #539

**Comment by cbalducci** [comment]
Sorry, i've got the internal gitlab email address and the github one mixed up. I've added the cbXXXX address to my github account, which is linked to my chef supermarket account where i've signed the CLA. If this doesn't work i will redo fork-commit-pr from scratch using the correct email address.
Cheers

P.S.
Could you please delete the email address from your previous comment? Thank you

**Pull request merged**